src/login/logind-gperf.gperf

```diff
@@ -30,10 +30,12 @@ Login.HandleHibernateKey,           config_parse_handle_action,         0, offse
 Login.HandleLidSwitch,              config_parse_handle_action,         0, offsetof(Manager, handle_lid_switch)
 Login.HandleLidSwitchExternalPower, config_parse_handle_action,         0, offsetof(Manager, handle_lid_switch_ep)
 Login.HandleLidSwitchDocked,        config_parse_handle_action,         0, offsetof(Manager, handle_lid_switch_docked)
+Login.HandleRestartKey,             config_parse_handle_action,         0, offsetof(Manager, handle_restart_key)
```

Comment:
so i think we should expose this as "RebootKey" rather than as "RestartKey". The underlying kernel name is just too confusing. We do similar clean-ups for the name for the suspend keys, i.e. KEY_SUSPEND we call "HibernateKey" and KEY_SLEEP we call "SuspendKey", because that's what they are used for, and the underlying kernel names are all confused...